cloud-controller-manager/do/loadbalancers.go

```diff
@@ -245,6 +249,13 @@ func (l *loadBalancers) GetLoadBalancerName(_ context.Context, clusterName strin
 }
 
 func getDefaultLoadBalancerName(service *v1.Service) string {
```

Comment:
TODO this is no longer a `Default` name, perhaps we should rename this function.